docs/_templates/changelog/v3.0.0.md

```diff
@@ -24,6 +24,12 @@
 
 - <abbr title="New Features">✨</abbr> **[#279](https://github.com/mcbookshelf/Bookshelf/issues/279)** - Added a `#bs.block:play_block_sound` tag for playing block sounds.
 
+### `🌱 bs.generate`
```

Comment:
i think it's best to put bs.generation here, also for our changelog splitter

Reply:
I don't think. People reading the changelog are searching for `bs.generate` since they don't know the module has been renamed. For me, this is starting from the next version we will change that.